docs/guides/contributor/setting-up-your-dev-env.md

```diff
@@ -30,7 +30,7 @@ nvm use 14
 
 ## Building locally
 
-To build locally, visit our Developer Workflows](workflows#local-checkout-build-artifacts-and-create-a-cluster) page.
+To build locally, visit our [Developer Workflows](workflows#local-checkout-build-artifacts-and-create-a-cluster) page.
```

Comment:
This may need an `.md` I think to be a working link and pass the link checker.

This might be a good example of what (I think) works to get proper relative internal linking: https://github.com/opstrace/opstrace/blame/06354ec27838204ddb5eaf170e75772a6c37869a/docs/guides/user/instrumenting-a-k8s-cluster.md#L19

Reply:
@jgehrcke fixed it, and it seems to work!

Reply:
I believe I fixed it - but the GitHub Action still seems to complain, but about a different link (If I see this correctly).